Rakefile

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env rake
 
+module TempFixForRakeLastComment
```

Comment:
Is that for the deprecation warning? If so, let's do this in a separate PR. It is also unclear if it belongs here or in the base framework (which would solve it for all plugins).